error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/SpringMvcAnnotation.java

```diff
@@ -114,9 +115,8 @@ private static Fix replaceAnnotation(
             .map(arg -> SourceCode.treeToString(arg, state))
             .collect(joining(", "));
 
-    return SuggestedFix.builder()
-        .addImport(ANN_PACKAGE_PREFIX + newAnnotation)
-        .replace(tree, String.format("@%s(%s)", newAnnotation, newArguments))
-        .build();
+    SuggestedFix.Builder fix = SuggestedFix.builder();
+    String annotation = SuggestedFixes.qualifyType(state, fix, ANN_PACKAGE_PREFIX + newAnnotation);
```

Comment:
To be consistent with namings of "replacement" variables inside the method.
```suggestion
    String newAnnotation = SuggestedFixes.qualifyType(state, fix, ANN_PACKAGE_PREFIX + annotation);
```

Reply:
This would require changing the method parameter name, which currently service as a bit of "self-documenting code" for the caller. At the same time, `annotation` is referenced only on the next line, which should make its semantics clear. So I'd prefer to keep it as-is.